llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -820,3 +820,12 @@ def WaveGetLaneIndex : DXILOp<111, waveGetLaneIndex> {
   let stages = [Stages<DXIL1_0, [all_stages]>];
   let attributes = [Attributes<DXIL1_0, [ReadNone]>];
 }
+
+def WaveAllBitCount : DXILOp<135, waveAllOp> {
```

Comment:
Initially, I thought this couldn't be right since the latter name `waveAllOp` mismatches the former `WaveAllBitCount` in more than just capitalization, but though DXIL.rst makes no mention of waveAllOp, that is the name of the actual DXIL op. Weird, but consistent.